ouroboros-consensus/changelog.d/js-rawlock.md

```diff
@@ -0,0 +1,3 @@
+## Non-breaking
+
+* Use [`rawlock`](https://hackage.haskell.org/package/rawlock) instead of the in-tree implementation.
```

Comment:
It is a breaking change according to PVP as we remove stuff, right?

Reply:
Hmmmm. Technically yes. But it comes from the `Util` subtree 😄 . But yeah, will change that once I push the hackage.nix update too